docs/dev.md

```diff
@@ -93,7 +93,18 @@ Your project must have a Dockerfile to use dev mode in container mode. A sample
 
 Note: Open Liberty images with the `kernel-slim` tag are not supported with dev mode at this time.
 
-Dev mode works with a temporary modified copy of your Dockerfile to allow for hot deployment during development. When dev mode starts up, it pulls the latest version of the parent image defined in the Dockerfile, builds the container image, then runs the container. Note that the context of the `docker build` command used to generate the container image is the directory containing the Dockerfile. When dev mode exits, the container is stopped and deleted, and the logs are preserved in the directory mentioned above.
+Dev mode works with a temporary, modified copy of your Dockerfile to allow for hot deployment during development as detailed below. When dev mode starts up, it pulls the latest version of the parent image defined in the Dockerfile, builds the container image, then runs the container. Note that the context of the `docker build` command used to generate the container image is the directory containing the Dockerfile. When dev mode exits, the container is stopped and deleted, and the logs are preserved in the directory mentioned above.
+
+Dev mode modifies the Dockerfile in order to allow for hot deployment. It only supports the application under development in the current project so it relies on the `.war.xml` file generated by the application build operation. This file is functionally equivalent to the `.war` file but it references class and resource files to pick up changes. To avoid application conflicts, dev mode removes all Dockerfile commands that copy or add a `.war` file. 
```

Comment:
Mention "loose application" so users can more easily look it up in the rest of the docs if they need to

Reply:
To be honest I was surprised the dev mode documentation hasn't mentioned that up to now.